en/components/general/wpf_to_angular_guide/two_way_binding.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Two-way data binding
+_description: WPF to Angular guide topics help you switch from WPF to Angular.
+_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular, two-way, binding
+---
+
+## Two-way data binding in Angular
+
+In the previous topic we discussed the [one-way binding](one_way_binding.md) in Angular, and how easy it was to use it. The one-way binding updates the view with data coming from the component class. Like in WPF, we can do the opposite operation - update the component class from the view. In that case we need to use a two-way binding and we will demonstrate its power and how easy it is to use it.
+
+Essentially the one-way binding is taking a state from the our component class and displaying it in our view. Let's look at this code:
```

Comment:
"taking a state from **the our** component class" - should remove the "the".

Reply:
Addressed